imp_act/environments/README.md

```diff
@@ -1,3 +1,34 @@
 ## Overview
 
+This directory is organized as follows:
+
 ![software architecture](https://github.com/user-attachments/assets/25639cfe-42d4-4fc3-8e89-b86482307364)
+
+### Wrappers
+
+<img src="wrappers.png" alt="wrappers" width="50%">
+
+#### Recorder Wrapper
+
+An optional wrapper to record the agent's interactions with the environment. This is useful for analyzing the agent's behavior. The Recoder wrapper is available in the `recorder.py` file, and stores the data in `tape` as dict.
+
+Usage:
+```python
+from imp_act.environments.recorder import Recoder
```

Comment:
Would be cool to include this in the main package init. (Similar to "make")

Reply:
that's a good idea, done! I also included RolloutPlotter.